web_monitoring/html_diff_render.py

```diff
@@ -1453,6 +1454,12 @@ def merge_change_groups(change_chunks, doc, tag_type=None):
 
 TagInfo = namedtuple('TagInfo', ('name', 'open', 'source'))
 
+@lru_cache(maxsize=1024)
+def tag_info(token):
```

Comment:
If we’re pulling this out of `reconcile_change_groups()`, would you please:

- Add a docstring
- Rename the `token` argument? (Maybe something like `tag` or `tag_source` or `tag_text` or `token_text` or `token_source` or `token_html`.)

Mainly this is because `lru_cache` would be unsafe to use with instances of `DiffToken` (two instances with different properties can still compare the same and wind up using the same slot in the cache’s underlying dictionary—you can test it to see).

I’m 99% certain we never wind up using a `DiffToken` instance as input to this function, but I want to make it extra clear so someone doesn’t accidentally do it. I actually started testing this because *I* forgot this doesn’t take `DiffToken` instances. :)

Reply:
Done!